src/components/ErrorModal.vue

```diff
@@ -0,0 +1,267 @@
+<template>
+  <div
+    class="error-modal"
+    @click="resolve"
+  >
+    <div class="modal-body" @click.stop>
+      <IconClose
+        class="close-modal"
+        @click="resolve"
+      />
+      <template v-if="step === 1">
+        <FailureIcon />
+        <h1>{{ $t('reportBug.Title') }}</h1>
+        <h2>{{ $t('reportBug.SubTitle') }}</h2>
+        <p class="description">
+          {{ $t('reportBug.Description') }}
+        </p>
+        <ButtonPlain
+          class="detailsButton"
+          @click="toggleDetails"
+        >
+          <IconEye />
+          {{ $t('reportBug.ViewDetails') }}
+        </ButtonPlain>
+        <p v-show="details">
+          {{ JSON.stringify(error) }}
+        </p>
+        <h2 class="reportTitle">
```

Comment:
```suggestion
        <h2 class="report-title">
```
usually, we are using kebab-case for class names, why you have switched to camelCase here and in other places?